endpoints/getting-started/openapi.yaml

```diff
@@ -6,6 +6,9 @@ info:
   version: "1.0.0"
 host: "echo-api.endpoints.YOUR-PROJECT-ID.cloud.goog"
 # [END swagger]
+# For App Engine deployments, delete the above "host:" line and remove the "# "
```

Comment:
Thanks @p-buse!

After reviewing the endpoints docs, I'm assuming that `host: "echo-api.endpoints.YOUR-PROJECT-ID.cloud.goog"` is not commented because it's used more often. Is that the case here?